spec/amp-html-format.md

```diff
@@ -331,6 +331,7 @@ Font providers can be whitelisted if they support CSS-only integrations and serv
 - Fonts.com: `https://fast.fonts.net`
 - Google Fonts: `https://fonts.googleapis.com`
 - Font Awesome: `https://maxcdn.bootstrapcdn.com`
+- [Typekit](https://blog.typekit.com/2017/08/30/css-font-loading/): `https://use.typekit.net/kitId.css` (replace `kitId` accordingly)
```

Comment:
This is probably a better link: https://helpx.adobe.com/typekit/using/google-amp.html

Reply:
agree, updated to https://helpx.adobe.com/typekit/using/google-amp.html